bytelatent/data/patcher.py

```diff
@@ -91,7 +91,7 @@ def calculate_entropies(
             split = split.reshape(-1, max_length)
             if device is not None:
                 split = split.to(device)
-            assert torch.all(split >= 0) and torch.all(split < 260)
+            # assert torch.all(split >= 0) and torch.all(split < 260)
```

Comment:
Why remove the assert? is this based on potentially not having just bytes and having other kinds of data?